src/pymodaq_plugins_thorlabs/daq_move_plugins/daq_move_KPZ101.py

```diff
@@ -0,0 +1,133 @@
+from pymodaq.control_modules.move_utility_classes import (
+    DAQ_Move_base, comon_parameters_fun, main, DataActuatorType, DataActuator)
+from pymodaq.utils.daq_utils import ThreadCommand
+from pymodaq.utils.parameter import Parameter
+from pymodaq_plugins_thorlabs.hardware.kinesis import serialnumbers_piezo, Piezo
+from pymodaq.utils.logger import set_logger, get_module_name
+
+logger = set_logger(get_module_name(__file__))
+
+
+class DAQ_Move_KPZ101(DAQ_Move_base):
+    """ Instrument plugin class for an actuator.
+
+    This object inherits all functionalities to communicate with PyMoDAQ’s DAQ_Move module through inheritance via
+    DAQ_Move_base. It makes a bridge between the DAQ_Move module and the Python wrapper of a particular instrument.
+
+    Attributes:
+    -----------
+    controller: object
+        The particular object that allow the communication with the hardware, in general a python wrapper around the
+         hardware library.
+
+    """
+    _controller_units = Piezo.default_units
+    is_multiaxes = True
+    _axes_names = {'1': 1}
+    _epsilon = 0.01
+    data_actuator_type = DataActuatorType.DataActuator
+    params = [
+                 {'title': 'Serial Number:', 'name': 'serial_number', 'type': 'list',
+                  'limits': serialnumbers_piezo, 'value': serialnumbers_piezo[0]}
+
+             ] + comon_parameters_fun(is_multiaxes, axes_names=_axes_names, epsilon=_epsilon)
+
+    def ini_attributes(self):
+        self.controller: Piezo = None
+        self._move_done = False
```

Comment:
what is this for? It seems it is not used? There is a boolean in DAQ_Move that deals with that feature, usually you don't have to set it!